test/clustertest/testplugin/TestPlugin.cpp

```diff
@@ -56,6 +58,32 @@ TestPluginCommand::TestPluginCommand(SQLiteCommand&& baseCommand, BedrockPlugin_
 {
 }
 
+TestPluginCommand::~TestPluginCommand()
+{
+    if (request.methodLine == "testescalate") {
+        string serverState = plugin().server.getState() == SQLiteNode::LEADING ? "LEADER" : "FOLLOWER";
```

Comment:
this line can incorrectly report the server's state, it assumes the server will always be leading or following. can't you do `plugin().server.stateName(plugin().server.getState())` or something similar (that implementation is kind of ugly) to get the actual state name as a string?